assets/search/actions.ts

```diff
@@ -22,6 +22,7 @@ import {
 } from './selectors';
 
 import {context} from 'selectors';
+import {fetchItems} from 'agenda/actions';
```

Comment:
Does this now not work in The Wire section? Originally it was calling the `wire/actions/fetchItems` and now its calling the `agenda/actions/fetchItems`. Both those 2 actions do different things.

We would need a better solution here that works for all sections, using dispatching `search/actions/toggleFilter` and `search/actions/setCreatedFilter`

Reply:
Maybe we should:
* remove the `fetchItems` from the `toggleFilter` & `setCreatedFilter` dispatch actions
* Add a call to `fetchItems` from where `toggleFilter` & `setCreatedFilter` actions are executed (filter panel etc)

Reply:
I checked wire and it's working but yes, I think we need a better solution to clearly define where the search is coming from, I'm just still thinking about it and I'll discuss it with Tomas

Reply:
I believe this needs to be fixed sooner rather than later, so we cannot wait for Tomas

Reply:
I'm not convinced this is the solution, as the `agenda/actions/fetchItems` runs a query against the Agenda API. So, when a filter is removed from the search results bar, a search against Agenda would happen even in the Wire page